client/src/App.js

```diff
@@ -89,7 +89,7 @@ function App() {
               </a>
             </div>
             <div>
-              <a href="#">
+              <a href="https://github.com/">
```

Comment:
Generally you want to link directly to the repository's code, and since we have that URL, we can put it in here!

```suggestion
              <a href="https://github.com/developer-delta/halogen">
```